nixos/roles/webproxy.nix

```diff
@@ -61,18 +61,59 @@ in
           command = "${cfg.package}/bin/varnishadm status";
           timeout = 180;
         };
-        varnish_http = {
+        varnish_http = let
+          check-varnish-http = pkgs.writers.writePython3 "check-varnish-http" {
+            flakeIgnore = [ "E501" ]; # ignore long lines
```

Comment:
i really really abhor this style of embedding here. if this is a separate standalone python script then it's much easier to achieve a proper feedback cycle during development. this is where code goes to die IMHO.

good enough for the fix now, it does make me wonder why we don't have better ready-made tooling at hand here. either we don't know nixpkgs well enough or there's something else fishy.

Reply:
Yes, I also considered moving this to a standalone file, but this would have required
1. another layer of wrapping to build a PATH with the executables we now instead just insert via string templating
2. having to look at yet another file at some different place to figure out that check script => fragmentation of code details